generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -173,7 +175,7 @@ nohttp:
 maven-compile:
     stage: build
     script:
-        - ./mvnw -ntp compile -Dmaven.repo.local=$MAVEN_USER_HOME
+        - ./mvnw compile -P-webpack -Dmaven.repo.local=$MAVEN_USER_HOME
```

Comment:
Did the `-ntp` got lost on purpose here?

Reply:
10x for catching this up 👍